packages/block-library/src/post-tags/index.php

```diff
@@ -20,11 +20,17 @@ function render_block_core_post_tags( $attributes, $content, $block ) {
 
 	$post_tags = get_the_tags( $block->context['postId'] );
 	if ( ! empty( $post_tags ) ) {
-		$output = '';
+		$output  = '';
```

Comment:
This is unnecessary, and we should also add the main block's class to the div to be consistent to other blocks (and also to give users a way to select it via CSS).

An example of how we did it for [other blocks](https://github.com/WordPress/gutenberg/blob/71bf1b5146d5dd0fc15792c1a98fc19df25b062f/packages/block-library/src/post-comments/index.php#L34-L39) (we could skip `$classes` but I presume we'll want to add the text align class eventually):

```php
$classes = 'wp-block-post-tags';
$output  = sprintf( '<div class="%1$s">', esc_attr( $classes ) );

// ...or just this for now:
$output = '<div class="wp-block-post-tags">';
```

Reply:
Good callout.

> I presume we'll want to add the text align class eventually
👍 I have an in-progress PR where I was planning to handle alignment https://github.com/WordPress/gutenberg/pull/24069.  I'll implement option #1.